src/Pages/ProductList/Components/ProductSale.jsx

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react';
+import './ProductSale.scss';
+
+class ProductSale extends Component {
+  render() {
+    const product = this.props.product;
```

Comment:
구조분해 할당은 아래와 같이 합니다 :)
```suggestion
    const {product} = this.props;
```

Reply:
넵 수정완료 했습니다. !! 감사합니다 :)